docs(appium): address appium 1.x uninstallation (#17999)
appium/appium: packages/appium/docs/en/guides/migrating-1-to-2.md

npm install --global appium --drivers=xcuitest,uiautomator2
```

> This will install Appium and the two drivers for you in one go. Please uninstall Appium 1.x npm package once if you get an error in the installation or Appium process failure. 

Reviewer: ```suggestion
This will install Appium and the two drivers for you in one go. Please uninstall any existing Appium 1.x npm packages (with `npm uninstall -g appium`) if you get an installation or startup error. 
```
